### ISSUE TITLE
Target size: missing elements referenced by aria-activedescendant

### ISSUE THREAD

**Issue by mfairchild365** [dequelabs/axe-core#4168]
### Product

axe-core

### Latest Version

- [X] I have tested the issue with the latest version of the product

### Issue Description

The rule for target size does not appear to handle elements referenced by `aria-activedescendant` well. For example, modify the options in the [scrollable listbox example](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-scrollable/) such that they fail this rule, run axe, and obverse that axe does not return any issues.

I haven't thought this through yet - I'm not sure how reasonable it is to automate this aspect, but I at least wanted to report it.

**Comment by straker** [comment]
Thanks for letting us know
